superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -113,6 +125,11 @@ export default function DataSourcePanel({
     columns,
     metrics,
   });
+  const [showAllMetrics, setShowAllMetrics] = useState(false);
+  const [showAllColumns, setShowAllColumns] = useState(false);
+
+  const DEFAULT_MAX_COLUMNS_LENGTH = 50;
```

Comment:
Should we put these constants in the superset config, then access these values from store of redux `explore.common.conf.DEFAULT_MAX_COLUMNS_LENGTH` and `explore.common.conf.DEFAULT_MAX_METRICS_LENGTH`